test/e2e/testdata/output-artifact-with-s3-bucket-creation-enabled.yaml

```diff
@@ -8,9 +8,29 @@ spec:
   templates:
     - name: main
       steps:
+        - - name: ensure-bucket-not-exists
+            template: ensure-bucket-not-exists
         - - name: generate-artifact
             template: whalesay
 
+    - name: ensure-bucket-not-exists
```

Comment:
this is flakey test, don't fix here, should be done elsewhere

Reply:
Elsewhere like a separate PR?

Reply:
yes

Reply:
I'll fix this , as it has been bugging me.